src/prpy/base/mico.py

```diff
@@ -99,4 +109,11 @@ def Servo(self, velocities):
             self.GetRobot().GetController().Reset(0)
             self.servo_simulator.SetVelocity(velocities)
         else:
-            raise NotImplementedError('Servo is not implemented.') 
+            self.SendVelocitiesToMico(velocities)
+            #reset watchdog timer
+            self.servo_watchdog.reset()
+
+
+    def SendVelocitiesToMico(self, velocities):
```

Comment:
I'd prefer to put this in `Servo`, since this code has a subset of that functionality: it doesn't work in simulation. Is there a strong reason to keep it as a separate function?